maven_iuvs/geometry.py

```diff
@@ -941,3 +941,178 @@ def pixelcorner_avg(pixel_x, pixel_y, pixel_z=None,
         return avgxy[:, :, 0], avgxy[:, :, 1], avgxy[:, :, 2]
 
     return avgxy[:, :, 0], avgxy[:, :, 1]
+
+
+def has_geometry_pvec(hdul):
+    """
+    Determines whether geodetic latitudes are available for the pixels in the pixel vector
+
+    Parameters
+    ----------
+    hdul : astropy FITS HDUList object
+           HDU list for a given observation
+
+    Returns
+    -------
+    n_int : int
+            number of integrations
+
+    """    
+    geom_quantity = hdul['PixelGeometry'].data['PIXEL_CORNER_LAT']
+    
+    n_nan = np.sum(np.isnan(geom_quantity))
+    n_quant = np.product(np.shape(geom_quantity))
+
+    nanfrac = n_nan / n_quant
+    
+    return (nanfrac < 1.0)
+
+
+def find_files_missing_geometry(file_index, show_total=False):
```

Comment:
echelle indexing files need to be specific to echelle

Reply:
Fixed by c361af4d4d7026605a880d550706d37d7c4378c2